test/parallel/test-tls-alert-handling.js

```diff
@@ -40,11 +40,13 @@ const errorHandler = common.mustCall((err) => {
     server.close();
 });
 const server = tls.createServer(opts, common.mustCall(function(s) {
+  console.log('server on secureConnection');
```

Comment:
nit: would prefer these to be comments rather than console.log statements

Reply:
Its not intended as docs, the docs already say the cb is attached to secureConnection. Debugging the tests is unreasonably hard without logging, every test that failed, the first thing I did was go through and log all the important events. The output isn't very verbose, and it doesn't even show up at all during normal node test runs. There are lots of console.log statements through the test suite (randomly, some have none, some have some). I can take them all out if you all want, but I feel bad for the next person who has to debug these, and will have to add them all back in locally.

Reply:
OK. I still generally don't like extraneous console output in tests but wont block on it.

Reply:
anyone else feel strongly? Its fast to delete them, slow to add them, so I'll do if some more people ask

Reply:
Fwiw, when I add `console.log()` to tests, it’s usually because I think it might be helpful for debugging a future flaky test failure that cannot be reproduced locally. I’m okay with keeping them if you think that that’s the case here.

Reply:
I don't want to side-track this PR with this,  I removed most of the console.log's that I added (a couple remain where they parallel the convention of the pre-existing test).